dash-renderer/src/actions/dependencies_ts.ts

```diff
@@ -14,20 +14,31 @@ import {
     pickBy,
     props,
     reduce,
-    zipObj
+    zipObj,
```

Comment:
Could we consider using `trailingComma: "none"`?

Reply:
Yeah I'm on board with no trailing commas - this way is designed to minimize diffs, but I agree that a bigger diff is justified in the name of better readability.